app/webapp.py

```
@@ -313,22 +361,31 @@ async def control_breeze_device(request: web.Request) -> web.Response:
     thermostat_swings = {sw.display: sw for sw in ThermostatSwing}
     body: dict = await request.json()
     try:
-        device_state = device_states.get(body.get(KEY_DEVICE_STATE))
-        thermostat_mode = thermostat_modes.get(body.get(KEY_THERMOSTAT_MODE))
-        target_temp = int(body[KEY_TARGET_TEMP]) if body.get(KEY_TARGET_TEMP) else None
-        fan_level = thermostat_fan_levels.get(body.get(KEY_FAN_LEVEL))
-        thermostat_swing = thermostat_swings.get(body.get(KEY_THERMOSTAT_SWING))
+        device_state = device_states.get(body.get(KEY_DEVICE_STATE, None))
+        thermostat_mode = thermostat_modes.get(body.get(KEY_THERMOSTAT_MODE, None))
+        target_temp = int(body[KEY_TARGET_TEMP]) if body.get(KEY_TARGET_TEMP) else 0
+        fan_level = thermostat_fan_levels.get(body.get(KEY_FAN_LEVEL, None))
+        thermostat_swing = thermostat_swings.get(body.get(KEY_THERMOSTAT_SWING, None))
```

What's the reason for the _None_ default addition? Are we handling _None_ for these properties? Because if not, it's best to not set a default and let the code throw an error when needed.

---

This is to match [this](https://github.com/TomerFi/aioswitcher/blob/dev/src/aioswitcher/api/__init__.py#L638-L642)
I can revert it back if you want..

---

As long as it's handled. I don't mind.